apiv2/tests.py

```diff
@@ -329,12 +331,14 @@ def test_num_queries(self):
             with self.assertNumQueries(1):
                 sounds_dict = Sound.objects.dict_ids(sound_ids=self.sids[0])
                 dummy_request = self.factory.get(reverse('apiv2-sound-text-search'), {'fields': field_set})
-                SoundListSerializer(list(sounds_dict.values())[0], context={'request': dummy_request})
+                # Call serializer .data to actually get the data and potentially trigger unwanted extra queries
+                _ = SoundListSerializer(list(sounds_dict.values())[0], context={'request': dummy_request}).data
```

Comment:
do you need `_ =`?

Reply:
Pycharm complains otherwise because the method is not assigned. I can remove it but I thought it was fine.

Reply:
I thought so. the alternative is to merge this test with the one that checks that the fields are in the returned data. no problem